lib/stsci/tools/bitmask.py

```diff
@@ -6,8 +6,8 @@
 import numpy as np
 from astropy.utils import deprecated
 
-__version__ = '1.1.1'
-__vdate__ = '30-January-2018'
+__version__ = '1.1.2'
+__vdate__ = '10-August-2021'
```

Comment:
To match the comments below.
```suggestion
__vdate__ = '11-August-2021'
```